### PR TITLE
webrtc-adapter version selector

### PR DESCRIPTION
**Pull request by CezaryDanielNowak**
This code is meant to help debugging regressions in adapter-webrtc.
User is able to select any version of the script available in cdnjs

example at the bottom: https://cezarydanielnowak.github.io/samples/src/content/getusermedia/gum/

### CODE REVIEW

**Comment by CezaryDanielNowak** [comment]
I signed it!

**Comment by KaptenJansson** [comment]
Have no idea why travis is not working for this PR

File: src/js/adapter-loader.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+(function() {
+  var adapterURL = localStorage.alternativeAdapterVersion
+    ? 'https://cdnjs.cloudflare.com/ajax/libs/webrtc-adapter/' + localStorage.alternativeAdapterVersion + '/adapter.min.js'
```
Comment (KaptenJansson) [inline src/js/adapter-loader.js:3]: Could you try to send a new commit and see if it triggers travis? You start by changing all the lines to be max 80 chars long (the travis ci will complain about this anyway when it actually runs).

**Review by KaptenJansson — CHANGES_REQUESTED** [review]

**Comment by fippo** [comment]
i'd do this without the UI. For dev purposes its ok to manipulate localStorage. Other than that this seems like a good idea

**Comment by KaptenJansson** [comment]
Ping

**Comment by KaptenJansson** [comment]
Pong

**Comment by KaptenJansson** [comment]
Closing due to age.

**Pull request closed without merging**